docs/architecture/adr/ADR045-dev-namespaces.md

```diff
@@ -0,0 +1,83 @@
+# ADR045: Dev namespaces
+
+## Status
+
+Pending
+
+## Context
+
+Following the retirement of gsp-local the service teams have no way of testing
+changes before a merge to master. So the feedback cycle is very slow, and
```

Comment:
This section should probably mention the problems of sandbox.

Reply:
I don't know what you mean by "the problems of sandbox"?

Reply:
The reasons we don't want people using the sandbox cluster for these activities.

Reply:
I've added some more info (https://github.com/alphagov/gsp/pull/842/commits/36969602abb6121750188530bd123ab19169ed32). Would you mind reviewing and let me know if that's sufficient?

Reply:
yep